snapshots/homicide/2024-10-30/unodc.xlsx.dvc

```diff
@@ -0,0 +1,45 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: United Nations Office on Drugs and Crime - Intentional Homicide Victims
+    description: |-
+      The United Nations Office on Drugs and Crime Intentional Homicide data are sourced from either criminal justice or public health systems. In the former, data are generated by law enforcement or criminal justice authorities in the process of recording and investigating a crime event, whereas in the latter, data are produced by health authorities certifying the cause of death of an individual. The criminal justice data was collected from national authorities with the annual United Nations Survey of Crime Trends and Operations of Criminal Justice Systems (UN-CTS). National focal points working in national agencies responsible for statistics on crime and the criminal justice system and nominated by the Permanent Mission to UNODC are responsible for compiling the data from the other relevant agencies before transmitting the UN-CTS to UNODC. Following the submission, UNODC checks for consistency and coherence with other data sources. Data on homicide from public health sources were primarily obtained from the WHO Mortality Database.10 This dataset is a comprehensive collection of mortality data by cause of death, sex, and age group conducted yearly by the WHO with Member States. Deaths coded with Internatioanl Classification of Disease (ICD10) codes X85-Y09 (injuries inflicted by another person with intent to injure or kill), and ICD10 code Y87.1 (sequelae of assault), generally correspond to the definition of intentional homicide. The population data used to calculate homicide rates is sourced from the World Population Prospect, Population Division, United Nations Department of Economic and Social Affairs.
```

Comment:
nitpicky but - I think this 10 was a footnote and can be deleted?

Reply:
Good catch!